tests/common/test_coordination.py

```diff
@@ -16,7 +16,7 @@ def test_group_uid(group: str):
 
 
 def test_type_checking_ignores_inject():
-    def example_function(x: Movable = inject("foo")) -> MsgGenerator:
+    def example_function(x: Movable = inject("foo")) -> MsgGenerator:  # noqa: B008
```

Comment:
[B008](https://docs.astral.sh/ruff/rules/function-call-in-default-argument/) is going to create some awkwardness for us, possibly even after #483 and associated work has been complete. I suggest raising another issue for dealing with it, unless that discussion has happened already.